static/app/components/forms/field/index.tsx

```diff
@@ -67,6 +67,11 @@ export interface FieldProps
    * User-facing field name
    */
   label?: React.ReactNode;
+  /**
+   * May be used to give the field an aria-label when the field's label is a
+   * react node.
+   */
+  labelText?: string;
```

Comment:
Had to make this a little more complicated. Some fields are a bit too fancy and use complicated react elements in their actual label. This is to help with that